target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -453,7 +453,7 @@ y1)
 	set_usb_led "$boardname:blue:usb"
 	ucidef_set_led_netdev "wifi" "WIFI" "$boardname:blue:wifi" "wlan1"
 	ucidef_set_led_netdev "wifi5g" "WIFI5G" "$boardname:blue:wifi5g" "wlan0"
-	ucidef_set_led_netdev "lan" "LAN" "$boardname:blue:lan" "eth0.1" "tx rx"
+	ucidef_set_led_switch "lan" "LAN" "$boardname:blue:lan" "switch0" "0x03"
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:blue:internet" "eth0.2" "tx rx"
```

Comment:
Would you please switch the internet led to the switchdev trigger as well. The netdev trigger will only work as long as VID 2 is used for wan which might not be the case in all configurations.

Reply:
There is a separated led for wan port which directly connected to ephy led pin and that LED could indicate the connection status for WAN port.I'm not sure whether I should keep the old configuration or just drop the configuration for internet LED at all.

Reply:
BTW in my memory the original firmware used an userspace program to turn on the internet led when there is a stable internet connection.